cli/flox/src/commands/mod.rs

```diff
@@ -350,7 +350,10 @@ impl FloxArgs {
         let system_user_name =
             std::env::var("USER").context("could not determine username from $USER")?;
         let system_hostname = sys_info::hostname().context("could not determine hostname")?;
-        let argv = std::env::args().collect();
+        let argv: Option<Vec<String>> = {
+            let args: Vec<String> = std::env::args().collect();
+            if args.is_empty() { None } else { Some(args) }
```

Comment:
I'm wondering if an empty vec isnt better than no command.
While both are unlikely, given that argv is provided by the system, at least we can assume a cli operation in the presence of a command value. `Some([])` is imo sstill closer to the thruth than None..